src/app/cli/src/client.ml

```diff
@@ -467,6 +470,26 @@ let send_payment =
   user_command body ~label:"payment" ~summary:"Send payment to an address"
     ~error:"Failed to send payment"
 
+let get_transaction_status =
+  Command.async ~summary:"Get the status of a transaction"
+    (Cli_lib.Background_daemon.init
+       Command.Param.(anon @@ ("txn" %: string))
+       ~f:(fun port serialized_transaction ->
+         match User_command.of_base58_check serialized_transaction with
+         | Ok user_command ->
+             dispatch_with_message Daemon_rpcs.Get_transaction_status.rpc
+               user_command port
+               ~success:
+                 (Or_error.map ~f:(fun status ->
+                      sprintf !"Transaction status:%s\n"
+                      @@ Transaction_status.State.to_string status ))
+               ~error:(fun e ->
+                 sprintf "Failed to get transaction status: %s"
+                   (Error.to_string_hum e) )
+         | Error _e ->
+             eprintf "Could not deserialize user command" ;
+             exit 1 ))
```

Comment:
We're keeping exit codes unique (and, specifically, non-1) for now to help with the "mysterious exit 1".  It looks like the next available exit code is 16.